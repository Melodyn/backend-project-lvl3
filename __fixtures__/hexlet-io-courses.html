<!DOCTYPE html>
<html lang="ru">
    <head>
        <meta charset="utf-8">
        <title>Курсы по программированию Хекслет</title>
        <link rel="shortcut icon" type="image/x-icon" href="https://cdn2.hexlet.io/assets/favicon.ico">
        <link rel="stylesheet" media="all" href="/assets/application.css" />
        <link href="/courses" rel="canonical">
    </head>
    <body>
        <img src="/assets/professions/nodejs.png" />
        <h3>
            <a href="/professions/frontend">Фронтенд-программист</a>
        </h3>
        <script src="https://js.stripe.com/v3/"></script>
        <script src="https://hexlet.io/packs/js/runtime.js"></script>
    </body>
</html>
