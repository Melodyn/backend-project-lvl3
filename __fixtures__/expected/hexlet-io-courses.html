<!DOCTYPE html><html lang="ru"><head>
<title>Курсы по программированию</title>
<link rel="shortcut icon" type="image/x-icon" href="https://cdn2.hexlet.io/assets/favicon-f5d8d6de8f142b12d62d657d0c3a49fd437282b92b0d418942f60657f79d805f.ico">
<link rel="stylesheet" media="all" href="hexlet-io-courses_files/hexlet-io-assets-application-8c09cda7aec4e387f473cc08d778b2a7f8a1e9bfe968af0633e6ab8c2f38e03f.css">
<link href="hexlet-io-courses_files/hexlet-io-courses.html" rel="canonical">
<meta charset="utf-8">
</head>
<body class="min-vh-100">
<div class="row row-cols-1 row-cols-sm-2 row-cols-lg-3">
<div class="col mb-4">
<div class="card h-100 shadow-sm x-shadow-fade-in">
<div class="card-body d-flex flex-column justify-content-between">
<img class="mb-4" width="50" height="50" alt="Иконка профессии Фронтенд-программист" src="hexlet-io-courses_files/hexlet-io-assets-professions-frontend-be958f979985faf47f82afea21e2d4f2ffb22b467f1c245d926dcb765b9ed953.svg">
<h3 class="h5 font-weight-medium mb-0">
<a class="x-link-without-decoration stretched-link" href="/professions/frontend">Фронтенд-программист</a>
</h3>
</div>
</div>
</div>
</div>
<script src="https://js.stripe.com/v3/"></script>
<script src="hexlet-io-courses_files/hexlet-io-packs-js-runtime-64630796b8e08f0f1f1d.js"></script>
</body></html>
